devops/github_issue_alert/autokitteh.yaml

```
@@ -0,0 +1,27 @@
+# This YAML file is a declarative manifest that describes the setup of
+# an AutoKitteh project that monitors comments on GitHub issues.
+
+version: v1
+
+project:
+  name: github_issue_alert
+
+  connections:
+    - name: slack_conn
+      integration: slack
+    - name: GitHub_conn
```

Also fix this in the triggers section below
```suggestion
    - name: github_conn
```